plugins/nanospell/plugin.js

```diff
@@ -582,6 +583,10 @@
 				}
 			}
 
+			function scheduleScanWords(event) {
+				setTimeout(scanWords.bind(this, event), 0);
```

Comment:
this one has very minimal benefit, since the scheduled method no longer actually does any write ops on the DOM.  but it can't hurt.